org.eclipse.xtext.common.types/model/JavaVMTypes.ecore

```diff
@@ -211,7 +211,7 @@
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="visibility" eType="#//JvmVisibility"/>
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="simpleName" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="identifier" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"
-        changeable="false" transient="true" derived="true"/>
+        transient="true"/>
```

Comment:
i assume the normal transient value handling will also handle this correct during serialization